### DISCUSSION TITLE
React plays Readme discussion

### DISCUSSION THREAD

**Discussion by hasnainmakada-99** [reactplay/react-play#55] [Q&A] [answered]
Tapas sir, I was assigned to make readme files for each of the play, so can I use any of the dependencies out there which makes an automatic readme or Do I have to manually make a readme for each play

- There are certain issues, In future if there are going to be more plays added to the site then it will be difficult to make readme files for each of the plays, so can I use any of the dependencies out there and it to the project

**Comment by atapas** [comment, marked as answer]
Hey @hasnainmakada-99 

The automation is already in place. We are using PlopJS which automates the file creation for us. You can take a look into the https://github.com/atapas/react-play/blob/main/plopfile.js file's action section to learn how are we creating some of the play files already.

Similarly, we need to create a `Readme.md` file under each of the play folders when we create the component for it. The Readme file should have only one Heading with the play name. Let me know if you have questions. Happy to answer.

**Reply by hasnainmakada-99** [reply]
Thank you sir

**Reply by hasnainmakada-99** [reply]
so sir if I manually do also, I have to create 1 readme file for each of the plays and the readme file should contain only a heading with the play name ?? Am I right ?

**Reply by atapas** [reply]
Yes, but as part of the current issue, let's not do it manually for existing plays. We can do the manual for existing plays in a separate issue. As part of this one, lets focus on the automatic creation of the readme file for a new play and test it using the `yarn create-play` command locally.

**Reply by hasnainmakada-99** [reply]
ok sir will try my best to solve this issue

**Reply by hasnainmakada-99** [reply]
ok sir will try my best to solve this issue

**Reply by atapas** [reply]
Sure, feel free to ask here if you need any help. Plop is a bit overwhelming to start with but a solid one to learn. All the best.

**Reply by hasnainmakada-99** [reply]
Thank you sir

**Reply by hasnainmakada-99** [reply]
Tapas sir, I've implemented the feature and its working perfectly, will you please review it I've already raised a PR.

**Reply by atapas** [reply]
Review done commented

**Reply by hasnainmakada-99** [reply]
Tapas sir can you please tell me how can we ramove our sample plays that we created

**Reply by atapas** [reply]
git rm file-name

git commit -m "message"

git push

**Reply by hasnainmakada-99** [reply]
ok sir I was concerned that the file name was passed in a lot of file so I think that it would throw error, Thank you for the help sir

**Reply by hasnainmakada-99** [reply]
tapas sir I am getting some conflicts regarding the yarn.lock file ? sir can you please tell me how do i resolve it

**Reply by atapas** [reply]
You need this to reset the file to same as main

https://stackoverflow.com/questions/37972753/git-reset-single-file-in-feature-branch-to-be-the-same-as-in-master

**Reply by hasnainmakada-99** [reply]
ok sir

**Reply by hasnainmakada-99** [reply]
sir i am closing this current pr which i raised and will again raise a new pr, is that okay sir

**Reply by atapas** [reply]
ok.

**Reply by hasnainmakada-99** [reply]
Tapas sir, I've raised the pr and in this one i've made sure that no yarn.lock & package.json file is added

**Reply by atapas** [reply]
@hasnainmakada-99 Don't commit any files related to sample play as we do not need them in production. We just need the hbs file and the plop js file.

**Reply by hasnainmakada-99** [reply]
ok sir, i've done it

**Reply by hasnainmakada-99** [reply]
please review it
